aioaws/core.py

```diff
@@ -32,6 +32,7 @@ def __init__(self, client: AsyncClient, config: 'BaseConfigProtocol', service: L
         self.client = client
         self.aws_access_key = get_config_attr(config, 'aws_access_key')
         self.aws_secret_key = get_config_attr(config, 'aws_secret_key')
+        self.aws_session_token = getattr(config, 'aws_session_token') if hasattr(config, 'aws_session_token') else ''
```

Comment:
```suggestion
        self.aws_session_token = getattr(config, 'aws_session_token', None)
```
Was there a reason to use an empty string as a default instead of `None`?

Reply:
The syntax in your suggestion is clearer, thank you. -> 7b7ba5f

It is possible to use `None` as a default here, but I used an empty string to keep the type signature consistent with `S3Config.aws_session_token` and `S3ConfigProtocol.aws_session_token`.